pkg/cri/opts/spec_linux.go

```diff
@@ -295,13 +319,22 @@ func WithDevices(osi osinterface.OS, config *runtime.ContainerConfig) oci.SpecOp
 			s.Linux.Resources = &runtimespec.LinuxResources{}
 		}
 
+		deviceOwnership := ""
+		if !disableDeviceOwnershipFromSecurityContext(config.GetAnnotations()) {
+			UID := getDeviceUID(config)
+			GID := getDeviceGID(config)
+			if UID > 0 && GID > 0 {
```

Comment:
UID=0 is technically valid. While most of callers won't specify 0, is it okay to reject zero here?

Reply:
good point let's unwind those new getDevice calls just do the code here rejecting the override for now only if not set?

Reply:
With UID/GID being zero the code falls back to what is available today, i.e., copy host UID/GID. Do you think we want to override those to zeros if user has specified `RunAsUser/RunAsGroup` as 0?

Reply:
I think we need to bring it up in the next call.. have a discussion regarding the default behavior change, this question and the others brought up to make sure there is no miss-understandings and document the decisions.

Reply:
@mikebrow OK, that works for me!